src/main/java/org/jenkinsci/plugins/github/util/JobInfoHelpers.java

```diff
@@ -141,5 +141,19 @@ protected Job asJob() {
             }
         };
     }
+
+    /**
+     * Can be useful to ignore child jobs on reregistering hooks
+     *
+     * @return predicate with true on apply if item is not a child of WorkflowMultiBranchProject
+     */
+    public static <ITEM extends Item> Predicate<ITEM> isNotChild() {
+        return new Predicate<ITEM>() {
+            public boolean apply(ITEM item) {
+                return !(item.getParent().getClass().getName().equals("org.jenkinsci.plugins.workflow.multibranch"
```

Comment:
`SCMSourceOwner`?

Reply:
Sorry, new to all the jenkins code.  Can you please elaborate on your statement?